src/migration.ts

```diff
@@ -135,23 +134,12 @@ export class Migration implements RunMigration {
         ? now.toISOString().replace(/\D/g, '')
         : now.valueOf();
 
-    const crossRequire = createRequire(
-      // @ts-expect-error: ignore until esm only
-      import.meta.url || __dirname
-    );
-    const moduleDir = dirname(
-      crossRequire.resolve('node-pg-migrate/package.json')
-    );
-
     const templateFileName =
       'templateFileName' in options
         ? resolve(process.cwd(), options.templateFileName)
         : resolve(
-            moduleDir,
-            join(
-              'templates',
-              `migration-template.${await resolveSuffix(directory, options)}`
-            )
+            'node_modules/node-pg-migrate/templates',
```

Comment:
**question (if-minor):** does this work in `pnpm` as well?  
why is this change better than the other?

Reply:
It does not depend on `tsup` handling `import.meta.url` and `__dirname`

Reply:
Good point, I will look into this later. I'm just only afraid of this code-piece in relation to several package-managers (yarn, pnpm, bun, and so on). e.g. yarn could have something so that there isn't even a `node_modules` folder, is it? (I'm not working personally with yarn anymore)